portals/devportal/src/main/webapp/source/src/app/components/Apis/Listing/ApiTagCloud.jsx

```diff
@@ -30,6 +30,7 @@ const PREFIX = 'ApiTagCloud';
 const classes = {
     clickablePointer: `${PREFIX}-clickablePointer`,
     filterTitle: `${PREFIX}-filterTitle`,
+    tagCloudTag: 'tag-cloud-tag',
```

Comment:
Let's prefix this class name with `PREFIX` as follows.

```js
tagCloudTag: `${PREFIX}-tag-cloud-tag`,
```

Reply:
BTW, is this CSS class used down there? I couldn't find a usage.

Reply:
It is inherited from the Tag Cloud library.